conf/views/chargeG/chargeDetails.njk

```diff
@@ -29,19 +30,19 @@
 
           {{ csrf() | safe }}
 
-          {{ govukInput({
-              id: "qropsReferenceNumber",
-              label: {
-                text: messages('chargeG.chargeDetails.qropsReferenceNumber.label'),
-                isPageHeading: false,
-                classes: "govuk-label--s"
-              },
-              name: "qropsReferenceNumber",
-              type: "text",
-              value: form.qropsReferenceNumber.value,
-              classes: "govuk-input--width-10",
-              errorMessage: form.qropsReferenceNumber.error
-            }) }}
+            {{ hmrcInputCurrency({
```

Comment:
This is a bit misleading since the QROPS number is not a currency field. Maybe rename the macro to something more generic, e.g. hmrcCustomInput